doc/interfaces.scd

```diff
@@ -70,6 +70,11 @@ the system will only respond to certain keywords by default:
 	is for backwards compatibility and should not be used in new
 	deployments.
 
+*pointopoint*
```

Comment:
This typo in original ifupdown drives me nuts.  It would be nice to remap this to `point-to-point`.

Reply:
I like it, done.